website/content/en/preview/faq.md

```diff
@@ -133,3 +133,24 @@ See [Deprovisioning nodes]({{< ref "./tasks/deprovisioning" >}}) for information
 ### How do I upgrade Karpenter?
 Karpenter is a controller that runs in your cluster, but it is not tied to a specific Kubernetes version, as the Cluster Autoscaler is.
 Use your existing upgrade mechanisms to upgrade your core add-ons in Kubernetes and keep Karpenter up to date on bug fixes and new features.
+
+### How to upgrade from previous version (before v0.6.1) to v0.6.2
+Karpenter v0.6.2 requires a new permission `ec2:DeleteLaunchTemplate` to clean up Launch Templates.
+The IAM Role used for IRSA is now named something predictable rather than eksctl's long IDs.
+To upgrade, please follow the following steps.
+```
+helm uninstall karpenter --namespace karpenter
```

Comment:
I don't think we want to restart the process. It should simply require:
1. Update the IAM permissions
2. Helm upgrade